### PR TITLE
added netbeans IDE setup

### PR DESCRIPTION
**Pull request by ankit-kumar-dwivedi**
@morevnaproject Please review :)

### CODE REVIEW

File: docs/common/Setting up IDE.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+The Apache Netbeans IDE is recommended for Synfig Studio developments. The Installer automatically prepares the IDE so that it comes with all required plug-ins, the correct workspace encoding settings, pre-configured code formatters and more. Simply follow these steps:
+
+    * Download the `Netbeans Installer. <https://netbeans.apache.org/download/nb111/nb111.html>`_
+    * Launch the Netbeans IDE and go to tools -> plugins.
+    * Go to Settings tab and enable **Netbeans 8.2 plugins portal** :
+        
+        .. image:: ../images/netbeans_plugin_settings.png
+
+    * Go to Available plugins and click on **check for newest**.
+    * Now select and install C/C++ plugin :
```
Comment (ice0) [inline docs/common/Setting up IDE.rst]: No need for space before ":"

File: docs/common/Setting up IDE.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        
+        .. image:: ../images/netbeans_cpp_plugin.png
+
+
+Setup New Synfig Project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+IDE Installation is now completed so let's configure our new synfig project.
+
+    * Click file -> new project 
+    * Select C/C++ and C/C++ Project with Existing Source :
```
Comment (ice0) [inline docs/common/Setting up IDE.rst]: No need for space before ":"

File: docs/common/Setting up IDE.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+        .. image:: ../images/netbeans_choose_project.png
+
+    * Select your synfig source file directory
+    * Under **Select configuration mode** select **custom** and press next :
```
Comment (ice0) [inline docs/common/Setting up IDE.rst]: No need for space before ":"

File: docs/common/Setting up IDE.rst [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+.. note::
+
+     **TL;DR**
+     You can also refer to this video tutorial -> https://youtu.be/op-PShrJTCM
+
+
+Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Apache Netbeans IDE is recommended for Synfig Studio developments. The Installer automatically prepares the IDE so that it comes with all required plug-ins, the correct workspace encoding settings, pre-configured code formatters and more. Simply follow these steps:
```
Comment (ice0) [inline docs/common/Setting up IDE.rst]: development

File: docs/common/Setting up IDE.rst [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+    * Download the `Netbeans Installer. <https://netbeans.apache.org/download/nb111/nb111.html>`_
```
Comment (ice0) [inline docs/common/Setting up IDE.rst]: I think "https://netbeans.apache.org/download/" will be enought.

File: docs/index.rst [hunk 6]
```diff
@@ -14,6 +14,7 @@ Welcome to Synfig developers documentation!
    :maxdepth: 2
    :caption: Contents:
 
+   common/Setting up IDE
```
Comment (ice0) [inline docs/index.rst]: I suggest to rename it to "Setting up Netbeans IDE", so we can add more IDE's in the future.

File: docs/common/Setting up IDE.rst [hunk 7]
```diff
@@ -0,0 +1,77 @@
+Setting up Netbeans IDE for Synfig
```
Comment (ice0) [inline docs/common/Setting up IDE.rst]: Setting up Netbeans IDE for Synfig development

File: docs/common/Setting up IDE.rst (hunk 5 above)
Comment (ankit-kumar-dwivedi) [reply docs/common/Setting up IDE.rst]: done

File: docs/index.rst (hunk 6 above)
Comment (ankit-kumar-dwivedi) [reply docs/index.rst]: Sure

**Comment by ankit-kumar-dwivedi** [comment]
@ice0 done changes

**Pull request merged**